app/models/jobsModel.js

```diff
@@ -1,22 +1,40 @@
 const db = require('../database/dbConfig.js')
+var crypto = require('crypto')
```

Comment:
since we will not be modifying the parameter in the model file, you can delete this line.